### PR TITLE
adds Offboarding Docs

### PR DESCRIPTION
**Pull request by themacexpert**
### Description

Adds Docs for Offboarding as a block producer. 

### Checklist

- [x] I have added a label to this PR 🏷️
- [x] I have run my changes through Grammarly
- [x] If this page requires a disclaimer, I have added one
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects

### CODE REVIEW

File: node-operators/block-producers/offboarding/offboarding.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+2. Select the **pooledStaking** module 
+3. Select the **requestUndelegate** extrinsic
+4. Enter your account, which is, again, the same account you are sending the extrinsic from and the account you want to decommission as a block producer
+5. Choose the target pool that you originally used when configuring your delegation (either Autocompounding or Manual)
+6. Select **Stake** from the dropdown
+7. Enter the amount to unstake. You'll need to submit the value in [Planck](https://wiki.polkadot.network/docs/learn-DOT#the-planck-unit){target=\_blank}. As a reminder, the minimum stake amount is {{ networks.dancebox.block_producers.min_self_del.dance }}. Therefore, if you delegated the minimum amount without autocompounding, you'll need to enter `{{ networks.dancebox.block_producers.min_self_del.planck }}`
+8. Click **Submit Transaction** and sign and send the transaction from your wallet
+
+![Create and submit an extrinsic to un-delegate on Polkadot.js Apps](/images/node-operators/block-producers/offboarding/offboarding/offboarding-1.webp)
+
```
[the remark this answers is not in the document]
Comment (eshaben) [reply node-operators/block-producers/offboarding/offboarding.md:28]: Just as a heads up, I recently got some feedback on the account setup page that I need to clarify a bit about the sessions and the delay so I have to do that yet - so if you use it as a reference, just try to make it a bit more clear 🙂

**Review by eshaben — CHANGES_REQUESTED** [review]
thanks for putting this together!! 🙌

File: node-operators/block-producers/offboarding/offboarding.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+2. Click the **+** button next to the extrinsic field
+3. A list of the eligible candidates and their stake will be displayed at the bottom of the page. You can search for your address to ensure that it does not exist in the list
+
+![Query the current list of eligible candidates on Polkadot.js Apps](/images/node-operators/block-producers/offboarding/offboarding/offboarding-5.webp)
+
+## Unmap Session Keys {: #unmap-session-keys }
+
+Session keys are used to perform network operations, such as signing blocks, whereas your block producer account holds the staked funds and has an on-chain identity. By unmapping the session key to your account, you sever the association between your block producer account and your block-producing node. 
+
+The unmapping step is taken only as part of the offboarding process. If you need to rotate / change your session keys, for example, a service provider runs a Tanssi block producer on your behalf and you're migrating to a new provider, you'll need to follow the instructions for mapping (rotating) your session keys. You should not purge your keys as part of the session key rotation process.
```
[the remark this answers is not in the document]
Comment (themacexpert) [reply node-operators/block-producers/offboarding/offboarding.md]: I changed this section up a bit but hadn't pushed the commit prior to the review

File: node-operators/block-producers/offboarding/offboarding.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+If at least two sessions have passed from the session you submitted the extrinsic, the request is ready to be executed. To do so, select **Extrinsics** from the **Developer** dropdown and take the following steps:
+
+1. Select the account from which you want to send the extrinsic
+2. Select the **pooledStaking** module
+3. Select the **executePendingOperations** extrinsic
+4. For **delegator**, enter your account, which is the same account you sent the self-delegate request from
+5. For **operation**, select **Leaving**
+6. For **candidate**, enter the same account as you did in the **delegator** field
+7. For **at**, enter the session key at which you submitted the delegate request
```
[the remark this answers is not in the document]
Comment (themacexpert) [reply node-operators/block-producers/offboarding/offboarding.md]: Yes, done!

File: node-operators/block-producers/offboarding/offboarding.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+When you set up your Tanssi block producer node, you had to submit a delegation bond of at least {{ networks.dancebox.block_producers.min_self_del.dance }} Dance. To get that back, and to remove your block producer from from the list of eligible candidates, you'll need to take a series of steps similiar to those contained in the onboarding process.
+
+Head to [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss://fraa-dancebox-rpc.a.dancebox.tanssi.network#/extrinsics){target=\_blank}, click on the **Developer** tab, select **Extrinsics** from the dropdown, and take the following steps:
+
+1. Select the account from which you want to send the extrinsic. This account must be your existing block producer account that you originally delegated from
+2. Select the **pooledStaking** module 
+3. Select the **requestUndelegate** extrinsic
+4. Enter your account, which is, again, the same account you are sending the extrinsic from and the account you want to decommission as a block producer
+5. Choose the target pool that you originally used when configuring your delegation (either Autocompounding or Manual)
+6. Select **Stake** from the dropdown
```
[the remark this answers is not in the document]
Comment (themacexpert) [reply node-operators/block-producers/offboarding/offboarding.md]: Done!

File: node-operators/block-producers/offboarding/.pages [hunk 5]
```diff
@@ -0,0 +1,5 @@
+title: Offboarding
+hide: false
+nav:
+  - index.md
+  - 'Decommission Block Producer': 'account.md'
```
Comment (eshaben) [inline node-operators/block-producers/offboarding/.pages:5]:
```suggestion
  - 'Decommission Your Account': 'account.md'
```

I wanted to put "Decommission Your Block Producer," so it sort of aligns with the rest of the titles, but thats too long -- it gets broken into two lines. So I'd prefer "Decommission Your Account" for the left nav

File: node-operators/block-producers/offboarding/account.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+
+### Viewing Existing Stake {: #viewing-existing-stake }
+
+Before undelegating, it is helpful first to see how much you have staked, as you'll need to provide this figure later. To do so, head to [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss://fraa-dancebox-rpc.a.dancebox.tanssi.network#/chainstate){target=\_blank}, click on the **Developer** tab, select **Chain State** from the dropdown, and take the following steps:
+
+1. Select the **pooledStaking** module
+2. Select the **pools** query
+3. Enter your block producer account
+4. Ensure that the **include option** slider is toggled on
+5. In the **option** field, you can select **CandidateTotalStake** to see the amount you have staked in Plank. Alternatively, you can select **AutoCompoundingShares** or **ManualRewardShares** depending on the staking configuration used when setting up your block producer node
```
Comment (eshaben) [inline node-operators/block-producers/offboarding/account.md:24]: i misled you on this; i'm sorry! 😅 😬  - its not `CandidateTotalStake`. I'm making sure I 100% understand this before getting back to you

Comment (eshaben) [reply node-operators/block-producers/offboarding/account.md:26]: gahhh, so there isn't a super straightforward way to do this with polkadot.js apps right now. users can query each of the following and add them: ManualRewardShares + AutoCompoundingShares + JoiningShares

so maybe for this guide, for step 5 use JoiiningShares and then after the image just tell them to repeat the steps and query ManualRewardShares and AutoCompoundingShares and add em up to get their self-del?

Comment (eshaben) [reply node-operators/block-producers/offboarding/account.md:26]: CandidateTotalStake includes their entire stake for all their delegations (not just their own) - shocking, i know! 🙃  haha i don't know how i made that mistake initially, my bad

File: node-operators/block-producers/offboarding/account.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+### Submit Undelegation Request {: #submit-undelegation-request }
+
+Head to [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss://fraa-dancebox-rpc.a.dancebox.tanssi.network#/extrinsics){target=\_blank}, click on the **Developer** tab, select **Extrinsics** from the dropdown, and take the following steps:
+
+1. Select the account from which you want to send the extrinsic. This account must be your existing block producer account that you initially delegated from
+2. Select the **pooledStaking** module 
+3. Select the **requestUndelegate** extrinsic
+4. Enter your account, which is, again, the same account you are sending the extrinsic from and the account you want to decommission as a block producer
+5. Choose the target pool that you originally used when configuring your delegation (either **Autocompounding** or **Manual**)
+6. Select **Stake** from the dropdown
```
Comment (eshaben) [inline node-operators/block-producers/offboarding/account.md:42]: i'd say switch this to shares

File: node-operators/block-producers/offboarding/account.md [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+7. Enter the amount to unstake. You'll need to submit the value in [Planck](https://wiki.polkadot.network/docs/learn-DOT#the-planck-unit){target=\_blank}. As a reminder, the minimum stake amount is {{ networks.dancebox.block_producers.min_self_del.dance }}. Therefore, if you delegated the minimum amount without autocompounding, you'll need to enter `{{ networks.dancebox.block_producers.min_self_del.planck }}`
```
Comment (eshaben) [inline node-operators/block-producers/offboarding/account.md:43]: we should switch this to shares

**Comment by themacexpert** [comment]
Thanks for the review @eshaben, fixed!

**Review by eshaben — APPROVED** [review]
Nice! LGTM! 🚢

**Pull request merged**